src/NuGet.Indexing/PerfEventTracker.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using NuGetGallery;
```

Comment:
So after looking at this, it looks like NuGet.Indexing has a dependency on NuGetGallery.Core for many things. This just opens up that reference to allow us to use the disposableAction here.